week3/aniketsaoji/anagrams.java

```diff
@@ -0,0 +1,44 @@
+package anagrams;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class anagrams {
+	
+	public static ArrayList<Character> returnList(String word) {
+		ArrayList<Character> wordList = new ArrayList<Character>();
+		for (int i = 0; i < word.length(); i++) {
+			wordList.add(word.charAt(i));
+		}
+		return wordList;
+	}
+	
+	public static void printAnagram(String word) throws FileNotFoundException {
+
+		File file = new File("wordsEn.txt");
+		Scanner scan = new Scanner(file);
+		while (scan.hasNextLine()) {
+			String comparedWord = scan.nextLine();
+			ArrayList<Character> comparedList = returnList(word);
+			if (word.length() != comparedWord.length()) 
```

Comment:
What about swapping this inequality test for an equality test. That way you can remove the `continue` and get rid of the `else` clause